$OpenBSD: patch-py_jk_lib_py,v 1.2 2012/06/15 13:50:58 gonzalo Exp $

Fix running jk_init trying to create a jail the first time

--- py/jk_lib.py.orig	Thu Jun  7 18:19:27 2012
+++ py/jk_lib.py	Thu Jun 14 11:51:39 2012
@@ -459,7 +459,10 @@ def create_parent_path(chroot,path,be_verbose=0, copy_
 		if (stat.S_ISDIR(sb.st_mode)):
 			if (be_verbose):
 				print 'Create directory '+jailpath
-			os.mkdir(jailpath, 0755)
+			try:
+				os.mkdir(jailpath, 0755)
+			except OSError, (errno,strerror):
+				sys.stderr.write('NOTE: Jail directory already existed:\n')
 			if (copy_permissions):
 				try:
 					copy_time_and_permissions(origpath, jailpath, be_verbose, allow_suid, copy_ownership)
